### PR TITLE
feat: New modules available: eventgrid domain, eventgrid system-topic, eventgrid  topic are now published.

### PR DESCRIPTION
**Pull request by fabmas**
## Description

New event-grid module migrated from CARML to AVM

| Pipeline |
| - |
| [avm.res.event-grid.domain](https://github.com/fabmas/bicep-registry-modules/actions/workflows/avm.res.event-grid.domain.yml) |
| [avm.res.event-grid.system-topic](https://github.com/fabmas/bicep-registry-modules/actions/workflows/avm.res.event-grid.system-topic.yml) |
| [avm.res.event-grid.topic](https://github.com/fabmas/bicep-registry-modules/actions/workflows/avm.res.event-grid.topic.yml) |

> [NOTE]: the 3 badges have an expected red status because they were tested in the main branch.

### CODE REVIEW

**Review by AlexanderSehr — COMMENTED** [review]
3 modules in one @fabmas , you're killing me :D

File: avm/res/batch/batch-account/main.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.22.6.54827",
```
[the remark this answers is not in the document]
Comment (fabmas) [reply avm/res/batch/batch-account/main.json:8]: Hi Alex, I've restored all the other modules from main except eventgrid.

[the remark this answers is not in the document]
Comment (AlexanderSehr) [reply avm/res/batch/batch-account/main.json:8]: Hey @fabmas, actually, now it showes 218 file changes :D

[the remark this answers is not in the document]
Comment (fabmas) [reply avm/res/batch/batch-account/main.json:8]: OK... I'll close this PR and I'll do 3 different PRs (one for each module) without modifying any other module

**Comment by fabmas** [comment]
Please ignore this PR... I'm gonna create 3 different pr (one for each module)

**Pull request closed without merging**